packages/p2p/src/stores/sendbird-store.js

```diff
@@ -237,7 +237,7 @@ export default class SendbirdStore extends BaseStore {
     markMessagesAsRead(should_check_scroll) {
         if (!this.active_chat_channel) return;
 
-        if (document.hasFocus()) {
+        if (document.hasFocus() && ((isMobile() && this.should_show_chat_modal) || isDesktop())) {
```

Comment:
Can we change the condition by checking messages_ref instead? Because in mobile view the ref should be null if chat modal is not visible.

So, like `if(document.hasFocus() && this.messages_ref)`